R/Model-methods.R

```diff
@@ -157,6 +206,56 @@ setMethod(
   }
 )
 
+## LogisticLogNormalOrdinal ----
+
+#' @describeIn doseFunction
```

Comment:
should this be 
```suggestion
#' @describeIn probFunction
```
?

Reply:
Yep.